backend/src/main/java/com/benine/backend/preset/Preset.java

```diff
@@ -157,6 +170,47 @@ public boolean equals(Object o) {
   }
 
   /**
+   * Creates an image for a preset.
+   * @param streamReader to get the snapShot from to save as image.
+   * @param folder to write the image to.
+   * @throws StreamNotAvailableException  If the camera does not have a stream.
+   * @throws IOException  If the image cannot be written.
+   * @throws SQLException if the image can not be saved in the database.
+   */
+  public void createImage(StreamReader streamReader, String folder) throws
+          StreamNotAvailableException, IOException, SQLException {
+
+    File path = getNewImagePath(folder);
+
+    VideoFrame snapShot = streamReader.getSnapShot();
+    MJPEGFrameResizer resizer = new MJPEGFrameResizer(160, 90);
```

Comment:
@jochem725 I've changed the image size to 640 x 360, but I have also added the image size to the config so it's now easy to adjust.